pkg/apis/experimental/types.go

```diff
@@ -461,3 +463,61 @@ type JobCondition struct {
 	// Human readable message indicating details about last transition.
 	Message string `json:"message,omitempty"`
 }
+
+// DerivedNodeMetricsList contains derived metrics for a single node. It holds
+// derived metrics in windows on the node itself and all of its system
```

Comment:
"about the node itself" would be clearer than "on the node itself" (which could be interpreted as: the metrics are physically stored on the node)